applications/GeoMechanicsApplication/tests/cpp_tests/test_solving_strategy_wrapper.cpp

```diff
@@ -232,6 +232,26 @@ KRATOS_TEST_CASE_IN_SUITE(SaveAndAccumulateTotalDisplacementField, KratosGeoMech
     KRATOS_EXPECT_EQ(p_node->GetSolutionStepValue(TOTAL_DISPLACEMENT), expected_total_displacement);
 }
 
+KRATOS_TEST_CASE_IN_SUITE(ComputeIncrementalDisplacementField, KratosGeoMechanicsFastSuite)
+{
+    Model model;
+    auto& model_part       = CreateDummyModelPart(model);
```

Comment:
Strictly speaking, we should prefix this name with `r_` to comply with the Kratos Style Guide. Having said that, I can imagine that this minor inconsistency appears in other tests in the same file as well. So we could either correct those as well, or do that in a separate PR. I'll leave that up to you.

Reply:
Done for this .cpp file.